src/pages/SettingsPage/UsersSettings/userDetail.jsx

```diff
@@ -280,6 +283,7 @@ const UserDetail = ({
         }).unwrap()
 
         toast.update(toastId.current, { render: `Updated user: ${user.name} ` })
+        dispatch(setUserData(data))
```

Comment:
This only updates the user's data and misses the attrib data like `fullName`. Look at ProfilePage.jsx

Reply:
Ok so this relates to first comment. I have updated user reducer names in `user.js` and their imports.
I have also added update for user attributes for userDetail. Do you agree with this approach ?

<img width="560" alt="image" src="https://github.com/ynput/ayon-frontend/assets/16327908/7a226316-c360-4bf4-a8e9-703a0202745f">

Reply:
looking at the guide from redux, maybe `userDataUpdate` and `userAttribsUpdate` would be better ? But maybe Im overthinking it...